docs/concepts/event-driven-architecture.md

```diff
@@ -61,17 +62,19 @@ For each service that needs to interact with the message broker, enable the mess
 
 ![Service Connections Tab](./assets/service-connections-tab.png)
 
-## Step 4: Install the Kafka Plugin
+## Step 4: Implement Event Production and Consumption
+
+In your producer services, add logic to publish events to the appropriate topics when certain actions or conditions occur. Use the provided Kafka client to send messages to the desired topics.
```

Comment:
IIUC this is the first time Kafka is mentioned in this article, and it some king of out-of-the-blue